workers/src/test/java/edu/cornell/testenv/testclasses/TestClassJUnit.java

```diff
@@ -0,0 +1,20 @@
+package edu.cornell.testenv.testclasses;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TestClassJUnit {
```

Comment:
Not sure if this can be moved to some example directory instead of the source code tree, but might be nicer to have this somewhere else